Forms/MainForm.cs

```diff
@@ -75,12 +78,22 @@ private void UpdateConfigurationSubstitutions()
         {
             Configuration.DefaultSubstitutions.Clear();
             Configuration.DefaultSubstitutions.AddRange(substitutionList1.Substitutions);
+            LoadConfigurationDetails();
         }
 
 
         private void LoadConfigurationDetails()
         {
-            txtWordList.Text = Configuration.DefaultWords.ToString();
+            string wordtext = Configuration.DefaultWords.ToString();
+            if (txtWordList.Text != null && txtWordList.Text != wordtext && txtWordList.Text != wordtext + " ")

```

Comment:
But if the wordList has the focus, any changes are directly applied to `Configuration.DefaultWords`?  we dont need to update the input with the same value?  The strength bar can still be updated?